client/src/Components/Work.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import '../Style/Work.css'
+import WorkBackground from '../Images/bright.JPEG'
+var sectionStyle = {
+    backgroundImage: `url(${WorkBackground})`,
+    backgroundPosition: 'center',
+    backgroundSize: 'cover',
+    backgroundRepeat: 'y',
+    width: '100%',
+    height: '100%',
+    minHeight:"95vh"
+};
+export default function Work(props) {
+    return (
+        <div style={sectionStyle} id="work-body">
+            <div id="work">
+                <br></br>
+           It's not where you've been, it's where you're going...
+           <br></br>
+            </div>
+            <div id="experience">
+                <p>Currently (and most importantly), I am working full time to complete a 12 week, 600 hour intensive web development bootcamp with instruction on HTML, CSS, Javascript, React, and databases supplemented by challenging weekly projects and culminating with a capstone project.
+                    </p>
+                <br></br>
+                <br></br>
+                <div id="code-skills-chart">
+                    <p>
+                        Languages&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;  React | JavaScript | HTML | CSS <br></br>
+                        Frameworks &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;            jQuery <br></br>
+                        Development&nbsp;&nbsp;&nbsp;&nbsp; Visual Studio <br></br>
+                        Photo Editing &nbsp;&nbsp;&nbsp;&nbsp;Adobe Lightroom | Photoshop <br></br>
+                        Source Control&nbsp;&nbsp;  Git | Github
+                    </p>
+                    <br></br>
+
+
+                </div>
```

Comment:
Adding extra width to this will help fix your issues with things wrapping to a new line, You could also break this div into a "right" and "left" column and do css individually onto each of them to make it exactly how you would want it to be done